package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "cd backend && npm run create-schema && cd .. && ./start.sh",
-    "postinstall": "cd backend && npm install && cd ../frontend && npm install && npm run build && cd ..",
-    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'"
+    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'",
```

Comment:
can't you simply ignore the node_modules altogether without listing the out in both directories?

Reply:
true, thank you, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/6de4130a5334e3f7cc77b41a9f0f53a38de58ed2